src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -245,9 +252,44 @@ public void setWebhookRegistered(boolean webhookRegistered) {
     protected void retrieve(@CheckForNull SCMSourceCriteria criteria, SCMHeadObserver observer,
                             @CheckForNull SCMHeadEvent<?> event,
                             TaskListener listener) throws IOException, InterruptedException {
+        if (event == null) {
+            //on manual scans & creation of jenkins jobs, we call out to bbs and fetch a list of open prs and
+            // sync up our local pullRequestStore
+            DescriptorImpl descriptor = (DescriptorImpl) getDescriptor();
```

Comment:
I think we should use the `TaskListener` to log some information to the user visible build log here.